File: app/src/main/java/com/mohamedtaha/imagine/ui/fragments/NotificationFragment.java
package com.mohamedtaha.imagine.ui.fragments;

import android.os.Bundle;

import androidx.preference.PreferenceFragmentCompat;

import com.mohamedtaha.imagine.R;


public class NotificationFragment extends PreferenceFragmentCompat{

        @Override
        public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
            setPreferencesFromResource(R.xml.notification_preferences, rootKey);

        }

}

File: app/src/main/java/com/mohamedtaha/imagine/ui/fragments/NotificationFragment.kt
package com.mohamedtaha.imagine.ui.fragments

import android.os.Bundle
import androidx.preference.PreferenceFragmentCompat
import com.mohamedtaha.imagine.R
import dagger.hilt.android.AndroidEntryPoint

@AndroidEntryPoint
class NotificationFragment : PreferenceFragmentCompat() {
    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
        setPreferencesFromResource(R.xml.notification_preferences, rootKey)
    }
}